buildroot/share/scripts/gen-tft-image.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+#
+# Marlin 3D Printer Firmware
+# Copyright (c) 2021 MarlinFirmware [https://github.com/MarlinFirmware/Marlin]
+#
+# Based on Sprinter and grbl.
+# Copyright (c) 2011 Camiel Gubbels / Erik van der Zalm
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+# Generate Marlin TFT Images from bitmaps/PNG/JPG
+
+import sys,re,struct
+from PIL import Image,ImageDraw
+
+def image2bin(image, output_file):
+	if '.c' in output_file:
```

Comment:
you probably meant `output_file.endswith('.c')` ?

Reply:
No, I meant `output_file.endswith(('.cpp', '.c'))`, but `'.c' in output_file` is the laziest way to do the same.